lib/typescript/render/components/RichText/index.tsx

```diff
@@ -0,0 +1,38 @@
+import React from 'react';
+import styles from './index.module.scss';
+import {Avatar} from '../Avatar';
+import {Message} from 'httpclient';
+import {DefaultMessageRenderingProps} from '..';
+
+type RichTextRenderProps = DefaultMessageRenderingProps & {
+  message: Message;
+};
+
+export const RichText = (props: RichTextRenderProps) => {
+  const {message, fromContact, sentAt, contact} = props;
+  const messageJSON = JSON.parse(message.content);
+  const messageText = messageJSON.text;
+  const messageFallback = messageJSON.fallback;
+  const messageIsRichText = messageJSON.containsRichText;
```

Comment:
All these values should be on the properties and the properties should be injected in the [render](https://github.com/airyhq/airy/pull/902/files#diff-c11b3c1a17a44fc8a03e882cce37071b72413f235bea2a7617b98d7b42629c85R15) function. Right now this UI component is aware of message content structure.